src/index.ts

```diff
@@ -820,6 +820,15 @@ chatbot_helper.setup(HandleMessage);
 // setup the twitch api
 await twitchApi.setup();
 
+if (twitchApi.tokenScopes.includes("channel:read:subscriptions")) {
+  // set up the eventsub listener for subs
+  twitchApi.esListener.onChannelSubscription(
+    twitchApi.broadcasterId,
+    twitch.handleSub
+  );
+  twitchApi.esListener.start();
+}
+
```

Comment:
I think we could also do this one in `setup()` of `twitch-api.ts`.
Maybe the `setup` function could be split up into separate tasks/functions.